src/game/stats.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "global/types.h"
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+void Stats_Init();
+void Stats_CalculateStats(int32_t uninit_item_count, FLOOR_INFO **floor_array);
```

Comment:
I think this can be `FLOOR_INFO*` and later on we just reduce one level of pointer dereferencing (e.g. remove `&` or change `->` to `.`)

Reply:
I don't think that will work. I think it's fine in the current form.